gatsby/src/pages/search.tsx

```diff
@@ -1,14 +1,278 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import { PageRendererProps, Link } from "gatsby";
+import parser from "react-html-parser";
+import { Card } from "@nice-digital/nds-card";
+import { SimplePagination } from "@nice-digital/nds-simple-pagination";
+import { Alert } from "@nice-digital/nds-alert";
+
 import { Layout } from "../components/Layout/Layout";
 import { SEO } from "../components/SEO/SEO";
 
-const SearchPage: React.FC = () => {
+type SearchPageProps = PageRendererProps;
+
+interface SearchResults {
+	failed: boolean;
+	resultCount: number;
+	firstResult: number;
+	lastResult: number;
+	finalSearchText: string;
+	finalSearchTextNoStopWords: string;
+	pageSize: number;
+	originalSearch: {
+		searchText: string;
+	};
+	pagerLinks: {
+		previous: {
+			fullUrl: string;
+		} | null;
+		next: {
+			fullUrl: string;
+		} | null;
+		pages: {
+			title: string;
+			url: {
+				fullUrl: string;
+			};
+			isCurrent: boolean;
+		}[];
+	};
+	documents: Document[];
+}
+
+type Document = {
+	id: string;
+	metaDescription: string;
+	contentId: string;
+	pathAndQuery: string;
+	sourceUrl: string;
+	teaser: string;
+	title: string;
+	url: string;
+};
+
+function titleString(text: string, page: number): string {
+	return text
```

Comment:
I find this really hard to read! I wonder if something like this is easier? Maybe it's just me?
```js
function titleString(searchText: string, pageIndex: number): string {
return [
  pageIndex > 1 && `Page ${pageIndex}`,
  searchText
].filter(Boolean).join(" | ")
}
```

Reply:
you know me i'd rather have 4 if statements...